src/System.Security.AccessControl/src/System/Security/AccessControl/ObjectSecurity.cs

```diff
@@ -0,0 +1,738 @@
+// ==++==
+// 
+//   Copyright (c) Microsoft Corporation.  All rights reserved.
+// 
+// ==--==
+/*============================================================
+**
+** Classes:  Object Security family of classes
+**
+**
+===========================================================*/
+
+using Microsoft.Win32;
+using System;
```

Comment:
Some usings can be eliminated, just do a pass at the end of cleanup to remove and sort usings in the project